src/site/setup.py

```diff
@@ -0,0 +1,51 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+
+# HISTORY.rst entry.
+VERSION = '1.0.0'
```

Comment:
please release a beta version for the first release. Such as "1.0.0b1"

Reply:
Changed it to 1.0.0b1, hoping this aligns with APEX criteria too

Reply:
For ignite, you can release a GA version before it. The first version must be Beta

Reply:
Got it, thanks